### ISSUE TITLE
FEATURE REQUEST

### ISSUE THREAD

**Issue by richxheat** [anujtenani/goaffpro#242]
One feature that GOAFFPRO is wholly lacking is the ability to customize the login page for affiliates. As of today, and more specifically, the login page for affiliates has no option to add an image at the top of the page. Therefore, the login page for affiliates is plain, unattractive,  and does not square or enhance the branding of the website. 

Moreover,  and oddly enough, the Sign-up page for affiliates have the options to add a banner at the top of the page. So, the million dollar question is, why doesn't the sign-up page have the same option? 

In short,  such a feature can be added in minutes ( I know this, since I'm a C# coder), and should be added to Goaffpro.

Have a great week!
